frontend/src/app/pages/about/template/template.component.ts

```diff
@@ -0,0 +1,51 @@
+import { Location as NgLocation } from '@angular/common';
+import { HttpClient } from '@angular/common/http';
+import type { OnDestroy, OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Subject, takeUntil } from 'rxjs';
+
+@Component({
+    selector: 'app-template',
+    templateUrl: './template.component.html',
+    styleUrls: ['./template.component.scss'],
+})
+export class TemplateComponent implements OnDestroy, OnInit {
```

Comment:
`TemplateComponent` is highly irritating in an angular context. https://angular.io/guide/architecture-components#introduction-to-components-and-templates

Reply:
Thanks for the hint. I've renamed it to `AboutPlaceholderComponent` in https://github.com/hpi-sam/digital-fuesim-manv/pull/643/commits/8bd302005949d818ed141cd282fb1dbbe65bde49